.github/workflows/build.yml

```diff
@@ -19,6 +23,112 @@ jobs:
           java-version: '11'
           java-package: jdk             
       - name: build and test     
-        run: ./gradlew test --init-script init.gradle
+        id: thebuild
```

Comment:
"thebuild" seems like a weird name.  I'm still new to gradle so I don't have any suggestions of what would be better.

Reply:
that id is the github action name, not the gradle task name. It's certainly awkward but I needed a specific name for the later step were it uses the environment variable this task outputs... I just couldn't think of a better one.